.github/workflows/kind2-ci.yml

```diff
@@ -17,9 +17,11 @@ jobs:
           - os: macos-10.15
             z3_install: brew install z3
             zmq_install: brew install zmq
+            ounit2_install: opam install ounit2
```

Comment:
If the command for installing ounit2 is the same for both operating systems, put the command where it is used without introducing an intermediate variable

Reply:
fixed